db/schema.rb

```diff
@@ -23,6 +23,19 @@
     t.datetime "updated_at"
   end
 
+  create_table "community_resources", force: true do |t|
```

Comment:
@gschorkopf I felt that `resource` was too generic of a name and opted for `community_resource` to be a bit more explicit.